biomage/experiment/utils.py

```diff
@@ -116,3 +118,35 @@ def download_S3_json(s3_obj, key, filepath):
     s3_obj.download_file(local_file)
 
     set_modified_date(file_location=local_file, date=s3_obj.last_modified)
+
+
+def get_cognito_username(email):
+    client = boto3.client("cognito-idp")
```

Comment:
[out of scope for this PR] Do you know if we can use this API and get the login times of a user? @aerlaut might be interested in looking into this for the user KPIs epic